vars/dockerExecuteOnKubernetes.groovy

```diff
@@ -416,6 +420,36 @@ private void unstashWorkspace(config, prefix) {
     }
 }
 
+private List getInitContainerList(config){
+    def initContainerSpecList = []
+    if (config.initContainerImage && config.containerMountPath) {
+        def initContainerName = config.initContainerImage.toLowerCase()
+        def initContainerSpec = [
+            name           : config.initContainerName,
+            image          : config.initContainerImage,
+            volumeMounts   : [[name: "volume", mountPath: config.containerMountPath]],
```

Comment:
What if somebody wants an initContainer without volumeMounts? This should be optional.